### PR TITLE
Rename templates to surfaces

### PR DESCRIPTION
**Pull request by PatrykSzwer**
#114

### CODE REVIEW

File: src/Website/Helpers/DataHelper.cs [hunk 1]
```diff
@@ -179,37 +179,5 @@ public void GenerateHolyGrailSurface()
             new WebMap() { Url = profileUrl, Section = content, ForeignUrl = "/userprofile", SortNumber = 2 };
             new WebMap() { Url = profileUrl, Section = footer, ForeignUrl = "/content/dynamic/userprofile/footer", SortNumber = 3 };
         }
-
-        public void GenerateLauncherSurface()
```
Comment (un-tone) [inline src/Website/Helpers/DataHelper.cs]: Please don't forget to remove (replace with another surface generation) same method from the test projects

File: src/Website/Website.csproj [hunk 2]
```diff
@@ -104,7 +104,7 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <Folder Include="wwwroot\sys\" />
+    <Folder Include="wwwroot\Website\sys\" />
```
Comment (un-tone) [inline src/Website/Website.csproj]: This folder must be removed from the project at all. Can you do it while you are here?

**Comment by PatrykSzwer** [comment]
Everything should be ok now. I also removed files/folders that You asked for.

**Comment by warpech** [comment]
@un-tone could you pls re-review?

**Comment by un-tone** [comment]
LGTM

**Pull request merged**